src/wgpu/wgpu-command.cpp

```
@@ -1059,6 +1061,14 @@ Result CommandBufferImpl::reset()
     return CommandBuffer::reset();
 }
 
+void CommandBufferImpl::setLabel(const char* label)
+{
+    if (label)
+    {
+        m_label = label;
+    }
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)